lib/bugsnag/middleware/delayed_job.rb

```diff
@@ -0,0 +1,71 @@
+module Bugsnag::Middleware
+  ##
+  # Attaches delayed_job information to an error report
+  class DelayedJob
+    def initialize(bugsnag)
+      @bugsnag = bugsnag
+    end
+
+    def call(report)
+      job = report.request_data[:delayed_job]
+      if job
+        job_data = {
+          :class => job.class.name,
+          :id => job.id
+        }
+        job_data[:priority] = job.priority if job.respond_to?(:priority)
+        job_data[:run_at] = job.run_at if job.respond_to?(:run_at)
+        job_data[:locked_at] = job.locked_at if job.respond_to?(:locked_at)
+        job_data[:locked_by] = job.locked_by if job.respond_to?(:locked_by)
+        job_data[:created_at] = job.created_at if job.respond_to?(:created_at)
+        job_data[:queue] = job.queue if job.respond_to?(:queue)
+
+        if job.respond_to?('payload_object')
```

Comment:
Use a Symbol for this and the argument to `respond_to?` on the next line (as it seems more conventional to use symbols as arguments to `respond_to?`)